app/presenters/hyrax/image_presenter.rb

```diff
@@ -0,0 +1,8 @@
+# Generated via
+#  `rails generate hyrax:work Image`
+module Hyrax
+  class ImagePresenter < Hyrax::WorkShowPresenter
+    # We do not use this generated ImagePresenter. Instead we use the
```

Comment:
Might be useful to keep it around, in case a later re-generation modifies it to keep up with hyrax.  If only to see the changes and evaluate whether they might also apply to anything substituted for this presenter.